File: mobile/src/main/java/com/chaos/databinding/services/GitHub.java
package com.chaos.databinding.services;

import com.chaos.databinding.models.Repo;
import com.chaos.databinding.models.User;

import retrofit.http.GET;
import retrofit.http.Path;
import rx.Observable;

public interface GitHub {

    @GET("/users/{user}")
    Observable<User> getUser(@Path("getUser") final String user);

    @GET("/users/{user}/repos")
    Observable<Repo[]> getRepos(@Path("user") final String user);
}

File: mobile/src/main/kotlin/io/ashdavies/databinding/services/GitHub.kt
package io.ashdavies.databinding.services

import io.ashdavies.databinding.models.Repo
import io.ashdavies.databinding.models.User
import io.reactivex.Single
import retrofit2.http.GET
import retrofit2.http.Path

internal interface GitHub {

  @GET("/users/{user}")
  fun getUser(@Path("user") user: String): Single<User>

  @GET("/users/{user}/repos")
  fun getRepos(@Path("user") user: String): Single<List<Repo>>
}
